Tests/RealTests/ElementaryFunctionChecks.swift

```diff
@@ -142,17 +147,20 @@ final class ElementaryFunctionChecks: XCTestCase {
   func testFloat() {
     Float.elementaryFunctionChecks()
     Float.realFunctionChecks()
+    Float.cosPiTests()
```

Comment:
The PR doesn't compile, all `cosPiTests` are missing

Reply:
Yep, it's a draft.